packages/strings/src/urls.ts

```diff
@@ -16,20 +16,24 @@ const safeJoinWithQuestionMark = safeJoin("?");
  */
 
 // Root is ignored on purpose
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export type UrlTokens<T extends string> = T extends `${infer Root}/:${infer Token}/${infer Rest}`
+// prettier-ignore
```

Comment:
What does it look like without ignoring?

Reply:
it moves the comments around so that they don't make sense with the lines I'm referencing